src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value, currantResult) {
+      return currantResult + value;
+    },
+    subtract(value, currantResult) {
+      return currantResult - value;
+    },
+    multiply(value, currantResult) {
+      return currantResult * value;
+    },
+    divide(value, currantResult) {
+      return currantResult / value;
```

Comment:
Consider adding a check to prevent division by zero, which would cause an error.